ISHPermissionKit/Requests/ISHPermissionRequestEventStore.m

```diff
@@ -55,7 +55,8 @@ - (void)requestUserPermissionWithCompletionBlock:(ISHPermissionRequestCompletion
     
     [self.eventStore requestAccessToEntityType:[self entityType] completion:^(BOOL granted, NSError *error) {
         dispatch_async(dispatch_get_main_queue(), ^{
-            completion(self, granted ? ISHPermissionStateAuthorized : ISHPermissionStateDenied, error);
+            NSError *externalError = (error.code == EKErrorEventStoreNotAuthorized) ? nil : error;
```

Comment:
would it? Assuming
a) error is not nil
-> it doesn't matter what `EKErrorEventStoreNotAuthorized` really is. Set to nil if equal, otherwise the error. Seems ok.
b) error is nil
-> no matter the value of `EKErrorEventStoreNotAuthorized, `error.code` will be `0` and it will always be nil.